File: fix.py
import sys

with open('mobile/lib/telas/painel_admin.dart', 'r', encoding='utf-8') as f:
    lines = f.readlines()

new_lines = []
skip = False
for i, line in enumerate(lines):
    if line.startswith('  void _showNotificacoesDialog() {') and i > 500:
        skip = True
    
    if not skip:
        new_lines.append(line)

    if skip and line.startswith('  }'):
        skip = False

with open('mobile/lib/telas/painel_admin.dart', 'w', encoding='utf-8') as f:
    f.writelines(new_lines)

File: fix.js
const fs = require('fs');

const path = 'mobile/lib/telas/painel_admin.dart';
const content = fs.readFileSync(path, 'utf8');
const lines = content.split(/\r?\n/);

const newLines = [];
let skip = false;

for (let i = 0; i < lines.length; i++) {
    if (lines[i].startsWith('  void _showNotificacoesDialog() {') && i > 500) {
        skip = true;
    }
    
    if (!skip) {
        newLines.push(lines[i]);
    }

    if (skip && lines[i].startsWith('  }')) {
        skip = false;
    }
}

fs.writeFileSync(path, newLines.join('\n'));
console.log('Fixed');
